sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -9152,6 +9198,7 @@ public void testCommaJoinTableLookupTableMismatchedTypes() throws Exception
         "SELECT COUNT(*)\n"
         + "FROM foo, lookup.lookyloo l, numfoo\n"
         + "WHERE foo.cnt = l.k AND l.k = numfoo.cnt\n",
+        queryContext,
```

Comment:
Should line 9219 use queryContext instead of QUERY_CONTEXT_DEFAULT?

Reply:
Same as above - when I made these changes I didn't actually dig in to how this was being verified. It's pretty inconsistent across the tests. Do you think it's ok if we remove references to the query context in another PR?

Reply:
Yeah, doesn't need to be addressed by this PR